UI-UX Design/README.md

```diff
@@ -276,12 +276,23 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
         <th>Description</th>
       </tr>
       <tr>
-      <td><a href="https://youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k&si=1cHzpetgv_-0uszp">Webdesign Interviews/Podcast</a></td>
-      <td>"Welcome to the 'Web Design Business Podcast Interviews' channel. On this channel, you'll find content covering the business aspects of web design, video interviews from podcast, and live stream Q&A sessions, with new videos typically posted 1-2 times a week.</td>
+        <td><a href="https://youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k&si=1cHzpetgv_-0uszp">Webdesign Interviews/Podcast</a></td>
+        <td>"Welcome to the 'Web Design Business Podcast Interviews' channel. On this channel, you'll find content covering the business aspects of web design, video interviews from podcast, and live stream Q&A sessions, with new videos typically posted 1-2 times a week.</td>
       </tr>
       <tr>
-      <td><a href="https://www.youtube.com/watch?v=2OzTi0iUMlE">UI/UX Design Job Interview Questions and Answers</a></td>
-      <td>Questions that you will most likely be asked in a UX design interview as well as going over tips on how best to answer them.</td>
+        <td><a href="https://www.youtube.com/watch?v=2OzTi0iUMlE">UI/UX Design Job Interview Questions and Answers</a></td>
+        <td>Questions that you will most likely be asked in a UX design interview as well as going over tips on how best to answer them.</td>
```

Comment:
Please remove this one, it exist already in the section. I will merged once you remove it. thanks

Reply:
Resolved with [74a3406](https://github.com/jfmartinz/ResourceHub/pull/108/commits/74a3406979749aeca13464bb71ea1f06bbac4794)